payroll/views/hr_payslip_line_views.xml

```diff
@@ -65,6 +78,7 @@
                 />
                 <field name="amount_select" />
                 <field name="slip_id" />
+                <field name="payslip_run_id" />
```

Comment:
Is there a case when the payslip_line_ids form is not displayed from the smart button in the payslip? There is no menu to access the full payslip_lines views. If i'm not wrong, then having a search for payslip_run_id will always return the same records in that tree view.

Reply:
The user may use the Action menu to navigate to payslip lines list view. Then the user can search.

I just want to see pivot totals for a batch. So I made an action menu for Payslip Batch.

Reply:
Great I didn't know that this action existed haha. Great